.github/workflows/cli-test.yml

```diff
@@ -47,6 +47,7 @@ jobs:
           PGPORT: 5432
           PGUSER: postgres
           PGDATABASE: postgres
+          PGSSLMODE: disable
```

Comment:
This would have been using default ssl not disable

Reply:
I'm not sure default can be used as sslmode https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-SSLMODE-STATEMENTS

Reply:
@BogdanAntoniu78 sorry I mean that by not specifying `PGSSLMODE` then it'll use default (which is usually `prefer`)